include/libwebsockets/lws-context-vhost.h

```diff
@@ -1031,13 +1031,23 @@ struct lws_protocol_vhost_options {
  * served from a filesystem, or it is a cgi etc.
  */
 enum lws_mount_protocols {
-	LWSMPRO_HTTP		= 0, /**< http reverse proxy */
-	LWSMPRO_HTTPS		= 1, /**< https reverse proxy */
-	LWSMPRO_FILE		= 2, /**< serve from filesystem directory */
-	LWSMPRO_CGI		= 3, /**< pass to CGI to handle */
-	LWSMPRO_REDIR_HTTP	= 4, /**< redirect to http:// url */
-	LWSMPRO_REDIR_HTTPS	= 5, /**< redirect to https:// url */
-	LWSMPRO_CALLBACK	= 6, /**< hand by named protocol's callback */
+	LWSMPRO_HTTP = 0, /**< http reverse proxy */
+	LWSMPRO_HTTPS = 1, /**< https reverse proxy */
+	LWSMPRO_FILE = 2, /**< serve from filesystem directory */
+	LWSMPRO_CGI = 3, /**< pass to CGI to handle */
+	LWSMPRO_REDIR_HTTP = 4, /**< redirect to http:// url */
+	LWSMPRO_REDIR_HTTPS = 5, /**< redirect to https:// url */
+	LWSMPRO_CALLBACK = 6, /**< hand by named protocol's callback */
+};
```

Comment:
This whitespace change was a mistake and I will undo it. Damn visual studio and its auto whitespace reformatting